lua/terrortown/lang/en.lua

```diff
@@ -1197,11 +1155,8 @@ L.hud_revival_time = "{time}s"
 L.door_destructible = "This door is destructible ({health}HP)."
 
 -- 2020-05-28
-L.confirm_detective_only = "Only detectives can confirm bodies."
-L.inspect_detective_only = "Only detectives can search bodies."
-L.corpse_hint_no_inspect = "Only detectives can search this body."
-L.corpse_hint_inspect_only = "Press [{usekey}] to search. Only detectives can confirm the body."
-L.corpse_hint_inspect_only_credits = "Press [{usekey}] to receive credits. Only detectives can search this body."
+L.corpse_hint_no_inspect = "Press [{usekey}] to search. [{walkkey} + {usekey}] to only view search UI."
+L.corpse_hint_inspect_only = "Press [{usekey}] to search. [{walkkey} + {usekey}] to only view search UI."
```

Comment:
You know, I wondered the same and decided to keep those seperated. Because they handle different situations and someone might want to reword them a bit. I was just unable to come up with a better sounding sentence. Hence I'd keep it that way